File: 0455-assign-cookies/0455-assign-cookies.cpp
class Solution {
public:
    int findContentChildren(vector<int>& g, vector<int>& s) {
        
       sort(g.begin(),g.end());   sort(s.begin(),s.end());
       int r=0,l=0;
       while(r<s.size() && l<g.size()){
         if(g[l]<=s[r]) l+=1;
         r++;
       }
       return l;
    }
};

File: 0455-assign-cookies/0455-assign-cookies.java
class Solution {
    public int findContentChildren(int[] g, int[] s) {
        Arrays.sort(g);
        Arrays.sort(s);
         int cnt = 0, idx = 0;
        int child = 0;
        int cookie = 0;

        while (child < g.length && cookie < s.length) {

            if (s[cookie] >= g[child]) {
                child++;
            }

            cookie++;
        }

        return child;
    }
}